Index: makefiles/Makefile.third_party.unix
===================================================================
--- makefiles/Makefile.third_party.unix	(revision 3867)
+++ makefiles/Makefile.third_party.unix	(working copy)
@@ -107,7 +107,7 @@
 	cd dependencies/sources/gflags-$(GFLAGS_TAG) && $(SET_PATH) ./autogen.sh
 
 dependencies/sources/gflags-$(GFLAGS_TAG)/autogen.sh:
-	svn co http://gflags.googlecode.com/svn/trunk/ -r $(GFLAGS_TAG) dependencies/sources/gflags-$(GFLAGS_TAG)
+	svn co http://gflags.googlecode.com/svn/trunk/ --non-interactive --trust-server-cert -r $(GFLAGS_TAG) dependencies/sources/gflags-$(GFLAGS_TAG)
 
 # Install protocol buffers.
 install_protobuf: dependencies/install/bin/protoc
@@ -122,13 +122,13 @@
 	cd dependencies/sources/protobuf-$(PROTOBUF_TAG) && $(SET_PATH) ./autogen.sh
 
 dependencies/sources/protobuf-$(PROTOBUF_TAG)/autogen.sh:
-	svn co http://protobuf.googlecode.com/svn/trunk/ -r $(PROTOBUF_TAG) dependencies/sources/protobuf-$(PROTOBUF_TAG)
+	svn co http://protobuf.googlecode.com/svn/trunk/ --non-interactive --trust-server-cert -r $(PROTOBUF_TAG) dependencies/sources/protobuf-$(PROTOBUF_TAG)
 
 # Intall Google Apputils Python.
 install_google_apputils: dependencies/sources/google_apputils_python-$(GOOGLE_APPUTILS_TAG)/README
 
 dependencies/sources/google_apputils_python-$(GOOGLE_APPUTILS_TAG)/README:
-	svn co http://google-apputils-python.googlecode.com/svn/trunk/ -r $(GOOGLE_APPUTILS_TAG) dependencies/sources/google_apputils_python-$(GOOGLE_APPUTILS_TAG)
+	svn co http://google-apputils-python.googlecode.com/svn/trunk/ --non-interactive --trust-server-cert -r $(GOOGLE_APPUTILS_TAG) dependencies/sources/google_apputils_python-$(GOOGLE_APPUTILS_TAG)
 
 # Install sparsehash.
 install_sparsehash: dependencies/install/include/google/dense_hash_map
@@ -143,7 +143,7 @@
 	cd dependencies/sources/sparsehash-$(SPARSEHASH_TAG) && $(SET_PATH) ./autogen.sh
 
 dependencies/sources/sparsehash-$(SPARSEHASH_TAG)/autogen.sh:
-	svn co http://sparsehash.googlecode.com/svn/trunk/ -r $(SPARSEHASH_TAG) dependencies/sources/sparsehash-$(SPARSEHASH_TAG)
+	svn co http://sparsehash.googlecode.com/svn/trunk/ --non-interactive --trust-server-cert -r $(SPARSEHASH_TAG) dependencies/sources/sparsehash-$(SPARSEHASH_TAG)
 	cd dependencies/sources/sparsehash-$(SPARSEHASH_TAG) && patch -p0 -i ../../archives/sparsehash.patch
 
 # Install Coin CBC.
@@ -156,7 +156,7 @@
 	cd dependencies/sources/cbc-$(CBC_TAG) && $(SET_PATH) ./configure --prefix=$(OR_ROOT_FULL)/dependencies/install --disable-bzlib --without-lapack --enable-static --enable-shared --with-pic
 
 dependencies/sources/cbc-$(CBC_TAG)/Makefile.in:
-	svn co https://projects.coin-or.org/svn/Cbc/releases/$(CBC_TAG) dependencies/sources/cbc-$(CBC_TAG)
+	svn co https://projects.coin-or.org/svn/Cbc/releases/$(CBC_TAG) --non-interactive --trust-server-cert dependencies/sources/cbc-$(CBC_TAG)
 
 # Install pcre (dependency of SWIG).
 dependencies/install/bin/pcretest: dependencies/sources/pcre-$(PCRE_TAG)/Makefile $(ACLOCAL_TARGET)
@@ -169,7 +169,7 @@
 	cd dependencies/sources/pcre-$(PCRE_TAG) && $(SET_PATH) ./autogen.sh
 
 dependencies/sources/pcre-$(PCRE_TAG)/autogen.sh:
-	svn co svn://vcs.exim.org/pcre/code/trunk -r $(PCRE_TAG) dependencies/sources/pcre-$(PCRE_TAG)
+	svn co svn://vcs.exim.org/pcre/code/trunk --non-interactive --trust-server-cert -r $(PCRE_TAG) dependencies/sources/pcre-$(PCRE_TAG)
 
 # Install SWIG.
 install_swig: dependencies/install/bin/swig
